src/midje/sweet.clj

```diff
@@ -220,6 +221,24 @@
   (pointer/set-fallback-line-number-from &form)
   (parse-tabular/parse (keys &env) &form))
 
+(defmacro for-all
+  "Check facts using values generated using test.check
+
+  Options
+  :seed       used to re-run previous checks
+  :max-size   controls the size of generated vlues
+  :num-tests  how many times to run the checks
+
+  (for-all 'name \"doc string\"
+    [pos-int gen/s-pos-int]
+    {:num-tests 10}
```

Comment:
What do people think about using this optional map to here to pass in (quick-check) configurations? 
Another way would be to bind dynamic variables, but it seems like more overhead (in terms of syntax written) for the user to configure `for-all`s by wrapping them in a `(binding [*seed* .. *num-tests* .. ] (for-all ..))`